Makefile

```diff
@@ -245,6 +245,34 @@ watch-front: ## Build front application and activate watch mode
 	@$(YARN) build-lti --watch
 .PHONY: watch-front
 
+clean-front-modules: ## Clean front application modules
+	rm -rf src/frontend/node_modules
+	rm -rf src/frontend/apps/lti_site/node_modules
+	rm -rf src/frontend/apps/standalone_site/node_modules
+	rm -rf src/frontend/packages/lib_classroom/node_modules
+	rm -rf src/frontend/packages/lib_common/node_modules
+	rm -rf src/frontend/packages/lib_components/node_modules
+	rm -rf src/frontend/packages/lib_tests/node_modules
+	rm -rf src/frontend/packages/lib_video/node_modules
+	rm -rf src/frontend/packages/lib_markdown/node_modules
+.PHONY: clean-front
+
+clean-front-libs: ## Clean front application libs
+	rm -fr src/frontend/packages/lib_classroom/lib/*
```

Comment:
can be replace with 

```suggestion
	rm -fr src/frontend/packages/lib_*/lib/*
```

WDYT ?

Reply:
Nice, I didn't think about this :facepalm:

Reply:
Me neiher, I thought I was clever using `find` and pipping with `xargs` 😅